### PR TITLE
PIR Sensor Documentation- fixes issue #200

### PR DESCRIPTION
**Pull request by SwindleA**
PIR sensor documentation

### CODE REVIEW

File: pi4micronaut-utils/src/docs/asciidoc/components/inputComponents/pirSensor.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+:rootpath: ../../
+endif::rootpath[]
+
+ifdef::rootpath[]
+:imagesdir: {rootpath}{imagesdir}
+endif::rootpath[]
+
+==== PIR Sensor
+[.text-right]
+https://github.com/oss-slu/Pi4Micronaut/edit/develop/micronautpi4j-utils/src/docs/asciidoc/components/inputComponents/pirSensor.adoc[Improve this doc]
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/components/inputComponents/pirSensor.adoc:13]: It's pointing to old one. Please check.

**Review by ruthvikm — CHANGES_REQUESTED** [review]

**Review by ruthvikm — APPROVED** [review]

File: components/src/main/java/com/opensourcewithslu/components/controllers/PIRSensorController.java [hunk 2]
```diff
@@ -11,33 +11,28 @@
...
     private final RGBLEDHelper rgbledHelper;
-
     /**
      * The PirSensorController constructor.
      * @param pirSensor A Pi4J DigitalInput object.
      * @param rgbLed A MultiPinConfiguration object.
      */
-    public PIRSensorController(@Named("pir-sensor")DigitalInput pirSensor,
-                               @Named("rgb-led-2") MultiPinConfiguration rgbLed) {
+    public PIRSensorController(@Named("pir-sensor")DigitalInput pirSensor,@Named("rgb-led-2") MultiPinConfiguration rgbLed) {
```
Comment (joefol) [inline components/src/main/java/com/opensourcewithslu/components/controllers/PIRSensorController.java:25]: Put a space between '@Named("pir-sensor") and DigitalInput and a space between the comma and '@Named()

**Review by adelgadoj3 — APPROVED** [review]
Looks good to me!

**Review by joefol — CHANGES_REQUESTED** [review]
Looks good

**Pull request merged**